x-pack/plugin/security/qa/jwt-realm/src/javaRestTest/java/org/elasticsearch/xpack/security/authc/jwt/JwtRestIT.java

```diff
@@ -81,6 +84,11 @@ public class JwtRestIT extends ESRestTestCase {
         ]}""".replaceAll("\\s", "");
     public static final String HMAC_PASSPHRASE = "test-HMAC/secret passphrase-value";
     private static final String VALID_SHARED_SECRET = "test-secret";
+    private static final MutableSettingsProvider keystoreSettings = new MutableSettingsProvider() {
```

Comment:
We should provide either a builder, or some static constructor for doing this kind of thing if it's a common pattern. Initialization blocks are kinda funky :)

I'll take that as a todo.

Reply:
Oh ya that'd be great. We use this pattern in a few places now.